src/db.rs

```diff
@@ -146,25 +146,65 @@ pub fn connection_url(url: &str) -> String {
 
 pub trait RequestTransaction {
     /// Obtain a read/write database connection from the primary pool
-    fn db_conn(&self) -> Result<DieselPooledConn<'_>, PoolError>;
+    fn db_write(&self) -> Result<DieselPooledConn<'_>, PoolError>;
 
     /// Obtain a readonly database connection from the replica pool
     ///
     /// If there is no replica pool, the primary pool is used instead.
```

Comment:
This needs to be updated to mention the primary pool is used even if the replica pool is unavailable.